javascript/src/main/components/ChannelMessages/MessageList.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import BootstrapTable from 'react-bootstrap-table-next';
+
+const userName = (userId, users) => {
+    let result = ""
+    for(let i = 0; i < users.length; i++) {
+        if(users[i].id == userId) {
+            result = users[i].real_name;
+        }
+    }
```

Comment:
Correct me if I'm wrong, but it looks like we're looping through the entire array of users for each message we have - that has O(n^2) complexity and might slow down the app if we have a lot of messages or users. Could we look into improving this, perhaps by storing the users in an object instead and then using the userId as a key?

Reply:
This was code that was merged in from someone else. I didn't work on MessageList.js.

Reply:
Andrew maybe we can add an issue for the W21 folks to address this?

Reply:
Opened issue #85